vaadin-platform-react-hybrid-security-test/src/main/frontend/views/@index.tsx

```diff
@@ -6,6 +6,7 @@ export const config: ViewConfig = {
     menu: {
         title: "root",
     },
+    lazy: false,
```

Comment:
workaround for https://github.com/vaadin/hilla/issues/3482. should be removed after.